api/lib/domain/validators/schooling-registration-validator.js

```diff
@@ -0,0 +1,70 @@
+const Joi = require('@hapi/joi').extend(require('@hapi/joi-date'));
+const { EntityValidationError } = require('../errors');
+
+const validationConfiguration = { allowUnknown: true };
+const MAX_LENGTH = 255;
+const CITY_CODE_LENGTH = 5;
+const PROVINCE_CODE_MIN_LENGTH = 2;
+const PROVINCE_CODE_MAX_LENGTH = 3;
+const COUNTRY_CODE_LENGTH = 5;
+const STATUSES = ['ST', 'AP'];
```

Comment:
```suggestion
const STATUSES = ['ST'];
```
Dans un premier temps, on ne pourra gérer que les élèves. Je propose donc d'enlever `AP` jusqu'au moment où on gérera les apprentis.

Reply:
@lisequesnel d'acc je corrige !